File: Ops.ViewModels/Ops/Medical.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OpsModels.Ops
{
    public class Medical
    {
        public int MedicalId { get; set; }
        public string DoctorName { get; set; }
        public HospitalOrClinic hospitalOrClinic { get; set; }
        public PrescriptionOrReport prescriptionOrReport { get; set; }
        public DateTime TreatementFrom { get; set; }
        public DateTime TreatementTo { get; set; }
        public string TreatementPurpose { get; set; }
        public string Description { get; set; }
        public int Status { get; set; }
    }
}

File: src/FormiginationUI/wwwroot/js/Ops/Profile/Medical.js
var MedicalManager = {
    saveMedical: function () {

        var MedicalObj = MedicalHelper.createMedicalObj();
        var jsonParam = 'MedicalObj=' + MedicalObj;
        var serviceUrl = "../Account/SaveMedical/";
        AjaxManager.SaveObject(serviceUrl, jsonParam, onSuccess);


        function onSuccess(jsonData) {
            if (jsonData == "Success") {

                AjaxManager.MsgBox('success', 'center', 'Success:', 'Medical Info Save/Update Successfully',
                    [{
                        addClass: 'btn btn-primary',
                        text: 'Ok',
                        onClick: function ($noty) {
                            $noty.close();

                        }
                    }]);
            }
        }


    }
};

var MedicalHelper = {
    init: function () {
        $("#btnMedicalSave").click(function () {
            MedicalManager.saveMedical();
        });
    },
    createMedicalObj: function () {
        var obj = new Object();
        obj.MedicalId = $("#txtMedicalId").val();
        obj.DoctorName = $("#txtDoctorName").val();
        obj.hospitalOrClinic = $("#txthospitalOrClinic").val();
        obj.prescriptionOrReport = $("#txtprescriptionOrReport").val();
        obj.TreatementFrom = $("#txtTreatementFrom").val();
        obj.TreatementTo = $("#txtTreatementTo").val();
        obj.TreatementPurpose = $("#txtTreatementPurpose").val();
        obj.Description = $("#txtDescription").val();
       
        return obj;
    },
};